api/api.js

```diff
@@ -1,7 +1,6 @@
 let path = require('path');
 let fs = require('fs');
 let _ = require('lodash');
-let YAML = require('yamljs');

```

Comment:
I slightly prefer yaml, but it annoys me at times with the spacing.  So yeah, I dont care, if you want to get rid of dependency, lets do it.